pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -81,6 +81,13 @@ func (c *Cluster) makeDeployment(mgrConfig *mgrConfig) *apps.Deployment {
 	}
 
 	replicas := int32(1)
+	if len(c.annotations) == 0 {
+		prometheusAnnotations := map[string]string{
+			"prometheus.io/scrape": "true",
+			"prometheus.io/port":   strconv.Itoa(metricsPort),
+		}
+		podSpec.ObjectMeta.Annotations = prometheusAnnotations
```

Comment:
Are you seeing the Prometheus annotations applied now when moving this here?

Reply:
yes, and test proves it

Reply:
Damn, `podSpec` isn't a pointer that is why the change later on isn't working. Nice for finding that!